projects/merlin-l2/index.js

```diff
@@ -36,20 +37,20 @@ module.exports = {
     tvl: sumTokensExport({
       ownerTokens: [
         [["0x7122985656e38BDC0302Db86685bb972b145bD3C"], "0x147A198d803D4a02b8bEc7CC78be1AbE0C3d93E5",], //sttone
-        [[ADDRESSES.ethereum.USDC], "0x8bb6cae3f1cada07dd14ba951e02886ea6bba183", //usdc
-        ]]
+        [[ADDRESSES.ethereum.USDC], "0x8bb6cae3f1cada07dd14ba951e02886ea6bba183",],//usdc
+        [["0x0000000000000000000000000000000000000000"],"0xC40329D3aE56Af6b0757C3fE53941DDCC3d92671",]] //eth
     }),
   },
   arbitrum: {
     tvl: sumTokensExport({
-      owners: ["0x8bb6cae3f1cada07dd14ba951e02886ea6bba183"],
+      owners: ["0x8bb6cae3f1cada07dd14ba951e02886ea6bba183","0xaf88d065e77c8cC2239327C5EDb3A432268e5831",],
```

Comment:
removing this as '0xaf88d065e77c8cC2239327C5EDb3A432268e5831' is USDC address and not your contract